playlist_souffle/definitions/playlist.py

```diff
@@ -0,0 +1,5 @@
+"""Module for defining the Playlist namedtuple"""
+
+from collections import namedtuple
+
+Playlist = namedtuple('Playlist', 'owner name tracks description')
```

Comment:
is `tracks` your `Track` or a spotify track id? is owner a username?

Reply:
Track is the track id. Shouldn't be spotify specific though, given that all music services have some id per track.

Owner is a user id. Think it should be owner_user_id? Or just user_id?

Reply:
Sorry was looking at wrong thing. track is the Track namedtuple

Reply:
if track is the Track namedtuple than I'd call username user_id